fpmsyncd/routesync.cpp

```diff
@@ -886,6 +1004,105 @@ bool RouteSync::getIfName(int if_index, char *if_name, size_t name_len)
     return true;
 }
 
+/*
+ * getNextHopList() - parses next hop list attached to route_obj
+ * @arg route_obj     (input) Netlink route object
+ * @arg gw_list       (output) comma-separated list of NH IP gateways
+ * @arg mpls_list     (output) comma-separated list of NH MPLS info
+ * @arg intf_list     (output) comma-separated list of NH interfaces
+ *
+ * Return bool true=Success false=Failure
+ */
+bool RouteSync::getNextHopList(struct rtnl_route *route_obj, string& gw_list,
```

Comment:
this function does not return false in any case. do we want to have return type ?

Reply:
Done.  Return value is changed to void.